Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -224,12 +56,16 @@ def validate_certificates_format():
             "The broker certificate seem to be incorrect as they don't end with '-----END CERTIFICATE-----'")
 
 
-def get_client_config(dxl_config_files: DXLConfigFiles) -> DxlClientConfig:
+def create_error_entry(contents):
+    return {'ContentsFormat': formats['text'], 'Type': entryTypes['error'], 'Contents': contents}
```

Comment:
why do we need this? this happens automatically when using return_error in the main.